api/src/paths/publish/attachment/resubmit.ts

```diff
@@ -0,0 +1,172 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { PROJECT_ROLE, SYSTEM_ROLE } from '../../../constants/roles';
+import { getDBConnection } from '../../../database/db';
+import { authorizeRequestHandler } from '../../../request-handlers/security/authorization';
+import { GCNotifyService, IgcNotifyRequestRemovalMessage } from '../../../services/gcnotify-service';
+import { getLogger } from '../../../utils/logger';
+
+const defaultLog = getLogger('/api/publish/attachment/resubmit');
+
+export const POST: Operation = [
+  authorizeRequestHandler((req) => {
+    return {
+      or: [
+        {
+          validSystemRoles: [SYSTEM_ROLE.PROJECT_CREATOR],
+          discriminator: 'SystemRole'
```

Comment:
Creator shouldn't be included here. The only actual thing a creator can do is create a new project. Everything else under a project is controlled by the project level roles.

We should include system admin and data admin though.